lib/signature.dart

```diff
@@ -100,7 +104,11 @@ class SignatureState extends State<Signature> {
       ),
       child: Listener(
         onPointerDown: (event) => _addPoint(event, PointType.tap),
-        onPointerUp: (event) => _addPoint(event, PointType.tap),
+        onPointerUp: (event){
+          _addPoint(event, PointType.tap);
+          //NOTIFY OF CHANGE AFTER MOVEMENT IS DONE
+          widget.onChanged(_points);
```

Comment:
IMHO onChanged should be optional as not everyone needs to use it. That means this line should use null-safe operator (`?.`)